tests/jeos/firstrun.pm

```diff
@@ -119,6 +119,24 @@ sub verify_partition_label {
     script_output('sfdisk -l') =~ m/Disklabel type:\s+$label/ or die "Wrong partion label found, expected '$label'";
 }
 
+sub verify_selinux {
+    if (has_selinux_by_default) {
+        # SELinux is default, should be enabled
+        validate_script_output("sestatus", sub { m/SELinux status:.*enabled/ });
+    } else {
+        # SELinux is not default, but might be supported
+        my $selinux_supported = script_run("grep -qw selinux /sys/kernel/security/lsm") == 0;
+        if ($selinux_supported) {
+            # supported, so it must be disabled
+            my $sestatus_available = assert_script_run("which sestatus", "SELinux is supported but 'sestatus' is not available");
```

Comment:
```suggestion
            assert_script_run("which sestatus", fail_message => "SELinux is supported but 'sestatus' is not available");
```

Reply:
Thanks, added in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20693/commits/f1aa1fc86b6998eaa0e5b2eab1ae0ae9ff1f0b6e